vmm/src/vm.rs

```diff
@@ -675,12 +675,21 @@ impl Vm {
         Ok(())
     }
 
-    fn os_signal_handler(signals: Signals, console_input_clone: Arc<Console>) {
+    fn os_signal_handler(signals: Signals, console_input_clone: Arc<Console>, on_tty: bool) {
         for signal in signals.forever() {
             if signal == SIGWINCH {
                 let (col, row) = get_win_size();
                 console_input_clone.update_console_size(col, row);
             }
+            if signal == SIGTERM || signal == SIGINT {
+                if on_tty {
+                    io::stdin()
+                        .lock()
+                        .set_canon_mode()
+                        .expect("failed to restore terminal mode");
+                }
+                std::process::exit((signal != SIGTERM) as i32);
+            }
```

Comment:
Could you please use a `match` for both `SIGWINCH` and `SIGTERM | SIGINT` as this is more idiomatic for Rust.

Reply:
ok